data.php

```diff
@@ -2,7 +2,7 @@
     $log = array();
     $divide =  parse_ini_file("/etc/pihole/setupVars.conf")['IPV6_ADDRESS'] != "" && parse_ini_file("/etc/pihole/setupVars.conf")['IPV4_ADDRESS'] != "";
     $hosts = file_exists("/etc/hosts") ? file("/etc/hosts") : array();
-    $log = new \SplFileObject('/var/log/pihole.log');
+    $log = getPiHoleLog();
```

Comment:
Did you mean to import system.php?
Edit: It probably gets imported in api.php when password.php is imported.